src/_guides/packages.md

```diff
@@ -0,0 +1,227 @@
+---
+title: How to use packages
+short-title: Packages
+description: Learn more about pub, Dart's tool for managing packages.
+---
+
+The Dart ecosystem uses _packages_ to manage shared software
+such as libraries and tools.
+To get Dart packages, you use the **pub package manager**.
+You can find publicly available packages on the [**Pub site**,]({{site.pub}})
+or you can load packages from the local file system or elsewhere,
+such as git repositories.
+Wherever your packages come from, pub manages version dependencies,
+helping you get package versions that work with each other and
+with your SDK version.
+
+Most [Dart-savvy IDEs][] offer support for using pub that
+includes creating, installing, updating, and publishing packages.
+Or you can use [`pub` on the command line](/tools/pub/cmd).
+
+{% include flutter-packages.md %}
```

Comment:
I think this Flutter "warning" stands out way too much. I would either totally remove it (those that do Flutter, are already using Flutter.dev), or move it single sentence like "If you are using Flutter, use `flutter pub <sub-command>` instead of `flutter <sub-command>`" up in the previous paragraph